extensions/components/com_redcore/site/language/da-DK/da-DK.com_redcore.ini

```diff
@@ -0,0 +1,400 @@
+; redCORE
+; Copyright (C) 2008 - 2016 redCOMPONENT.com. All rights reserved.
+; License http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL, see LICENSE.php
+; Note : All ini files need to be saved as UTF-8
+
+COM_REDCORE="redCORE - komponent"
+COM_REDCORE_DESC="redCORE"
+
+JNEW="Ny"
+JEDIT="Ret"
+; COM_REDCORE_CONFIGURE="Configure"
+COM_REDCORE_SAVE_SUCCESS="Gemt"
+COM_REDCORE_UPDATE="Opdater"
+COM_REDCORE_NOTHING_TO_DISPLAY="Der er intet at vise"
+; COM_REDCORE_N_ITEMS_DELETED="%d items successfully deleted"
+; COM_REDCORE_N_ITEMS_DELETED_1="%d item successfully deleted"
+; COM_REDCORE_N_ITEMS_PUBLISHED="%d items successfully published"
+; COM_REDCORE_N_ITEMS_PUBLISHED_1="%d item successfully published"
+; COM_REDCORE_N_ITEMS_UNPUBLISHED="%d items successfully unpublished"
+; COM_REDCORE_N_ITEMS_UNPUBLISHED_1="%d item successfully unpublished"
+; COM_REDCORE_DOCUMENTATION_LINK_TITLE="Documentation"
+; COM_REDCORE_PLEASE_SELECT_ITEM="Please select item."
+
+; Config
+COM_REDCORE_CONFIG_FORM_TITLE="Konfiguration"
+COM_REDCORE_CONFIG_MAIN_COMPONENT_CONFIGURATION="Hovedkonfiguration"
+COM_REDCORE_CONFIG_MAIN_COMPONENT_CONFIGURATION_DESC="Indstil konfigurationsparametre for applikation."
+COM_REDCORE_TRANSLATIONS="Oversættelser"
+COM_REDCORE_TRANSLATIONS_DESC="Installer og administrer oversættelse indholdselement XML-filer."
+COM_REDCORE_CONFIG_MAIN_COMPONENT_INFO="Applikation info"
+COM_REDCORE_CONFIG_PHP_VERSION="PHP version"
+COM_REDCORE_CONFIG_MYSQL_VERSION="MySQL version"
+COM_REDCORE_CONFIG_EXTENSION_SUPPORTED="Understøttet"
+COM_REDCORE_CONFIG_WRITABLE="Skrivbar"
+COM_REDCORE_CONFIG_MODULES="Moduler"
+COM_REDCORE_CONFIG_PLUGINS="Programudvidelser"
+COM_REDCORE_CONFIG_NAME="Navn"
+COM_REDCORE_CONFIG_TRANSLATIONS_CONTENT_ELEMENT_NO_FILES_AVAILABLE="Der er ingen indholdselement filer installeret til din udvidelse."
+COM_REDCORE_CONFIG_TRANSLATIONS_CONTENT_ELEMENT_INSTALLED="Indholdselement installeret."
+COM_REDCORE_CONFIG_TRANSLATIONS_CONTENT_ELEMENT_UNINSTALLED="Indholdselement afinstalleret."
```

Comment:
You're absolutely right. I think the integration to the frontend went a bit too quickly, especially in regards to the language files (as is evident by the volume of comments regarding those).